src/status_im2/contexts/chat/lightbox/zoomable_image/view.cljs

```diff
@@ -191,64 +205,75 @@
+           curr-orientation                        (or (rf/sub [:lightbox/orientation])
+                                                       orientation/portrait)
+           portrait?                               (= curr-orientation orientation/portrait)
+           dimensions                              (utils/get-dimensions image-width
+                                                                         image-height
+                                                                         curr-orientation
+                                                                         args)
+           animations                              {:scale         (anim/use-val c/min-scale)
+                                                    :saved-scale   (anim/use-val c/min-scale)
+                                                    :pan-x-start   (anim/use-val c/init-offset)
+                                                    :pan-x         (anim/use-val c/init-offset)
+                                                    :pan-y-start   (anim/use-val c/init-offset)
+                                                    :pan-y         (anim/use-val c/init-offset)
+                                                    :pinch-x-start (anim/use-val c/init-offset)
+                                                    :pinch-x       (anim/use-val c/init-offset)
+                                                    :pinch-y-start (anim/use-val c/init-offset)
+                                                    :pinch-y       (anim/use-val c/init-offset)
+                                                    :pinch-x-max   (anim/use-val js/Infinity)
+                                                    :pinch-y-max   (anim/use-val js/Infinity)
+                                                    :rotate        (anim/use-val c/init-rotation)
+                                                    :rotate-scale  (anim/use-val c/min-scale)}
+           props                                   {:pan-x-enabled? (reagent/atom false)
+                                                    :pan-y-enabled? (reagent/atom false)
+                                                    :focal-x        (reagent/atom nil)
+                                                    :focal-y        (reagent/atom nil)}
+           rescale                                 (fn [value exit?]
+                                                     (utils/rescale-image value
+                                                                          exit?
+                                                                          dimensions
+                                                                          animations
+                                                                          props))]
+       (rn/use-effect-once (fn []
+                             (js/setTimeout #(reset! set-full-height? true) 500)
+                             js/undefined))
```

Comment:
It seems `use-effect-once` uses `use-effect`, which already handles the `js/undefined` case. You can probably remove this.